docs/spec/spec.md

```diff
@@ -327,16 +356,19 @@ Required permissions: `reports.read`.
 
 ## GET /reports/:name/versions/:version/all/
 
-Downloads a zip file of everything (including data).
+Downloads a zip file of all files associated with a report version (including data).
```

Comment:
With `reports.read` this returns only artefacts, resources, and the readme (I think! double check code); with `reports.review` it's all files.